pkg/apis/machine/v1alpha1/types.go

```diff
@@ -742,6 +742,7 @@ type OpenStackMachineClassSpec struct {
 	Networks         []OpenStackNetwork      `json:"networks,omitempty"`
 	SecretRef        *corev1.SecretReference `json:"secretRef,omitempty"`
 	PodNetworkCidr   string                  `json:"podNetworkCidr"`
+	RootDiskSize     int                     `json:"rootDiskSize,omitempty"` // in GB
```

Comment:
Why `resource.Quantity` is not used?
g/g uses `resource.Quantity` for `volume.size` in the Shoot spec - ref https://github.com/gardener/gardener/blob/ab4ddfc16c0709b0d301ee8713f61fad7fcbeb6f/pkg/apis/core/v1beta1/types_cloudprofile.go#L136-L137